angular/src/directives/control-value-accessors/text-value-accessor.ts

```diff
@@ -35,6 +35,7 @@ export class TextValueAccessorDirective extends ValueAccessor {
     },
   ],
 })
+// TODO rename this value accessor to `TextValueAccessorDirective` when search-bar is updated
```

Comment:
How come? Wouldn't we just get rid of the `TextValueAccessorDirective` class/directive altogether and be done?

Reply:
Removing `TextValueAccessorDirective` would introduce a breaking change to the Angular package, which we don't need to make. The "new" value accessor only exists in the span of migrating the controls to using `ionInput` instead of `ionChange`, then it can assume the existing name.